plugins/run.go

```diff
@@ -132,6 +132,12 @@ func runPlugin(plugin Plugin, args []string, pluginInput PluginInput) error {
 		return err
 	}
 
+	var payload []byte
```

Comment:
instead, I suggest to modify the `runPlugin` method -> `func runPlugin(plugin Plugin, args []string, envs map[string]string, input []byte) error`.

current `pluginInput` contains configuration for the plugin run -> env
input would be the data to be sent on the plugin command's stdin

it would make clear what input threated how, this change will need you to modify `RunPluginByEvent` as well, to distinguish plugin configs (envs) and the payload (input).

Reply:
I am not sure if we should differentiate between envs and stdin here. We should discuss this